### ISSUE TITLE
memtable flush controller too aggressive after conversion to CPU Scheduler

### ISSUE THREAD

**Issue by glommer** [scylladb/scylla#3342]
In the original CPU quota based memtable flush controller, we had the following parameters:
```
    static constexpr float q1 = 0.01;
    static constexpr float q2 = 0.2;
    static constexpr float qmax = 1;

```

When we moved to the CPU scheduler, we made q1 equal to 100, which is roughly speaking 10 % of the CPU for the cases in which we are disputing with just requests. It is way too high: When we are below the soft limit, we want to flush very slowly. If we keep flushing too fast we may end the flush too soon and disrupt the foreground load more than we wanted to.

Testing a SMP=2 update heavy workload, I can see the request/s graph a lot bumpier than it used to be.

**Comment by slivne** [comment]
how can we reproduce this via cassandra-stress  ?

**Comment by avikivity** [comment]
Patch on list

**Comment by avikivity** [comment]
Fixed by 80651e6dccff4869a9bcedd08e6b90bb0e0779c2.
